app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -1001,6 +1005,10 @@ object TelemetryWrapper {
 
         EventBuilder(Category.ACTION, Method.ADD, Object.TAB, Value.TOPSITE)
                 .queue()
+
+        if (AFFILIATE_SITES.contains(source)) {
```

Comment:
Please add a comment like "source could be other values" or "we only care about this two source"..

Or maybe just 
```
if (!!TextUtils.isEmpty(source)) {
     AdjustHelper.trackEvent(EVENT_CLICK_AFFILIATE_LINK)
}

```

Reply:
Fixed